common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -549,6 +549,45 @@ public int findInSet(UTF8String match) {
     return 0;
   }
 
+  public int findInSet(UTF8String match, int collationId) {
```

Comment:
I think we now have a lot of new methods 
consider adding comments such as: https://github.com/apache/spark/pull/45791/files#r1549415554